README.md

```diff
@@ -8,7 +8,7 @@ speech recognition APIs.
 > This SDK only supports hosted usage of api.deepgram.com.
 
 To access the API you will need a Deepgram account. Sign up for free at
-[signup][signup].
+[signup][https://console.deepgram.com/signup?utm_source=dotnet-sdk&utm_content=readme].
```

Comment:
Couple things:

- This link was in the footer. Should we update it there or just remove it from the footer.
- Let's change the UTM/URL to: https://console.deepgram.com/signup?utm_medium=github&utm_source=DEVREL&utm_content=dotnet-sdk